crates/eips/Cargo.toml

```diff
@@ -12,8 +12,12 @@ repository.workspace = true
 exclude.workspace = true
```

Comment:
these changes appear non-functional. please remove them :)

Reply:
done, sorry. they happened coincidentally in the process.